### DISCUSSION TITLE
Multiple Urls Support

### DISCUSSION THREAD

**Discussion by llucmk** [keepassxreboot/keepassxc#10060] [Ideas]
I'm not sure if this has been suggested already, I was not able to find anything alike.

It seems that it would be a nice feature to have multiple urls available for entries. Multiple websites contain different url's for login or going from A to B but being in the same service. A feature to allow multiple url's same way as the tag works would be useful to avoid extensions or password fillers to detect properly the service you are running and auto fill passwords.

**Comment by droidmonkey** [comment]
https://keepassxc.org/docs/KeePassXC_UserGuide#_advanced_usage

**Reply by laur89** [reply]
Is there a good reason why glob/regex patterns aren't supported? Bit annoying all the sub-domains need to be manually configured. E.g. need to define
```
http://example.com
http://domain1.example.com
http://domain2.example.com
```

instead of
```
http://example.com
http://*.example.com
```

or why not even `http://(\w+\.)?example.com`

**Reply by droidmonkey** [reply]
Having simply `http://example.com` covers all your examples. We match everything below the highest subdomain specified.

**Reply by laur89** [reply]
Any idea approx. how long has this been the case?

**Reply by droidmonkey** [reply]
Since the browser capability was introduced, 7 years ago

**Reply by laur89** [reply]
Odd. Configured one domain ~2y back and then defining subdomains was necessary. That site has since changed the login mechanism so I'm unable to reproduce past few months. Will remember to try using single domain again next time.

**Reply by droidmonkey** [reply]
Since you wrote `http://` be sure your site isn't actually `https://`. We are strict about that.
